app/serializers/search/postgres_comment_serializer.rb

```diff
@@ -0,0 +1,41 @@
+module Search
+  # TODO[@atsmith813]: Rename this to CommentSerializer once Elasticsearch is removed
+  class PostgresCommentSerializer < ApplicationSerializer
+    attribute :id, &:search_id
+
+    attributes :path, :public_reactions_count
+
+    attribute :body_text, &:body_markdown
+
+    attribute :class_name do |_comment, _params|
+      "Comment"
+    end
+
+    attribute :highlight do |comment, _params|
+      {
+        body_text: [comment.pg_search_highlight]
+      }
+    rescue PgSearch::PgSearchHighlightNotSelected
+      # This is needed because in Search::Postgres::Comment we only call the
+      # search if a term is provided. This means if a user searches with a
+      # blank term, we skip the line that executes .with_pg_search_highlight.
+      # Skipping this AND trying to call .pg_search_highlight raises an error
+      # which we ignore here - basically filling in highlights if they're
+      # there.
+    end
+
+    attribute :hotness_score, &:score
+    attribute :published, &:commentable_published
```

Comment:
Since we filter by `articles.published = true`, this can simply be `true`:

```ruby
    attribute :published, -> { true }
```

Reply:
This comes at the cost of being less flexible to future changes (i.e. the filter is removed from the service and we'd have to remember to explicitly change the serializer). But I'm down to send it. I'll change it.

Reply:
I don't think we'll have a filter on draft comments though, comments can't be draft. I think this exists only because `FeedContent` in ES merges various types of data